python/srf/_pysrf/include/pysrf/port_builders.hpp

```diff
@@ -1,6 +1,6 @@
 /**
- * SPDX-FileCopyrightText: Copyright (c) 2021-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
- * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Copyright (c) 2021-2022, NVIDIA CORPORATION &
+ * AFFILIATES. All rights reserved. SPDX-License-Identifier: Apache-2.0
```

Comment:
did something happen to your formatter?

Reply:
Not sure. I'll run the formatters just to be sure. But CI didnt have any issues with it.